unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/GltfContainer/GltfContainerHandler.cs

```diff
@@ -366,42 +366,42 @@ void RemoveActiveColliders(IList<Collider> colliders, IInternalECSComponent<Inte
 
                 if ((colliderLayer & LAYER_PHYSICS) != 0)
                 {
-                    physicColliders ??= new InternalColliders();
-                    physicColliders.colliders.Add(collider, colliderLayer);
+                    physicColliders ??= new InternalColliders(new KeyValueSet<Collider, uint>());
```

Comment:
here as well - is it possible to cache, clear and re-use instead of creating new instances?